lib/sensly/sensors/mq_sensor.rb

```diff
@@ -0,0 +1,56 @@
+require 'sensly/sensors/base_sensor'
+
+module Sensly
+  class MQSensor < BaseSensor
+    ##
+    # Value of the load reistor in Ohms.
+    ##
+    R_LOAD = 10_000.0
+
+    ##
+    #
+    ##
+    def initialize(adc_value, r0, temperature, humidity)
+      super adc_value
+
+      @r0 = float r0
+      @temperature = float temperature
+      @humidity = float humidity
+    end
+
+    ##
+    # Convert a raw ADC value to resistance of the sensor.
+    ##
+    def adc_to_rs
```

Comment:
@samAlti can you check the docs for `adc_to_rs` and `rs_r0_ratio` (below) are correct?